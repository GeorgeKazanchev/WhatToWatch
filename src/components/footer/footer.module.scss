$dark-color: #54503e;

.copyright {
    text-align: center;
    font-size: 14px;
    line-height: 16px;
    color: $dark-color;

    p {
        margin: 0;
    }
}
