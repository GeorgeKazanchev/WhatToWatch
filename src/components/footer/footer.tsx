import React from 'react';
import * as styles from './footer.module.scss';
import { Logo } from '../logo/logo';

type Props = {
    isMainPage?: boolean,
};

export const Footer: React.FC<Props> = ({ isMainPage = false }) => {
    return (
        <footer>
            <Logo
                isMainPage={isMainPage}
                isLight={true}
            />
            <div className={styles.copyright}>
                <p>Â© 2024 What to watch Ltd.</p>
            </div>
        </footer>
    );
};
