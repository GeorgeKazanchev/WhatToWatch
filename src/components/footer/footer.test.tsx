import React from 'react';
import renderer from 'react-test-renderer';
import { describe, it, expect } from '@jest/globals';
import { Footer } from './footer';

describe('Footer', () => {
    it('Correctly renders', () => {
        const tree = renderer
            .create(
                <Footer
                    isMainPage={true}
                />
            )
            .toJSON();

        expect(tree).toMatchSnapshot();
    });
});
