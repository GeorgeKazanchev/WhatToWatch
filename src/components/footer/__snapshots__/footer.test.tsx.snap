// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Correctly renders 1`] = `
<footer>
  <div>
    <a
      className="undefined undefined"
    >
      <span>
        W
      </span>
      <span>
        T
      </span>
      <span>
        W
      </span>
    </a>
  </div>
  <div>
    <p>
      Â© 2024 What to watch Ltd.
    </p>
  </div>
</footer>
`;
